Spitfire/webrtc.props

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="PropertySheets" />
+  <PropertyGroup Label="UserMacros" >
+    <WebRtcSrc>$(ProjectDir)..\webrtc-checkout\src</WebRtcSrc>
+    <WebRtcOut>$(WebRtcSrc)\out\$(Platform)\$(Configuration)</WebRtcOut>
+  </PropertyGroup>
+  <PropertyGroup />
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(WebRtcSrc)\third_party\abseil-cpp;$(WebRtcSrc)</AdditionalIncludeDirectories>
+    </ClCompile>
+    <Link>
+      <AdditionalLibraryDirectories>%(AdditionalLibraryDirectories);$(WebRtcOut)\obj</AdditionalLibraryDirectories>
+      <AdditionalDependencies>%(AdditionalDependencies);webrtc.lib</AdditionalDependencies>
```

Comment:
Why are we changing library loading here when `Spitfire.vcxproj` already contains ` <AdditionalDependencies>webrtc.lib;winmm.lib; Secur32.lib;%(AdditionalDependencies)</AdditionalDependencies>`

Reply:
I cleared settings references nd combined them in single [.props file](https://github.com/RainwayApp/spitfire/blob/CORE-41/Spitfire/webrtc.props) where they are better visible including both .h and .lib.